python_websocket_server/README.md

```diff
@@ -70,3 +70,9 @@ DeepSpeech service. The websocket timeout on the ingress is set to 1 hour.
 ## Contributing
 
 Bug reports and merge requests are welcome.
+
+### Running tests
+
+```
+python -m pytest tests/test_app.py
```

Comment:
This looks good, but it's not integrated with out current CI setup, so it's never going to run. Are you fine with that, or do you plan on integrating later?

Reply:
I'm planning to integrate later, is that fine?

As an aside: At work, I have a GitLab CI/CD pipeline to build and test this code. Based on our recent discussions about GitLab CI, do you think this repo might be a good (easy) starting point to explore GitLab CI? Since this repo only contains examples, all of which I'm guessing can run in unix Docker containers, it should be fairly easy to write a CI/CD pipeline specifically for this repo.

Reply:
> I'm planning to integrate later, is that fine? (Created #93 to track.)

Yes


> As an aside: At work, I have a GitLab CI/CD pipeline to build and test this code. Based on our recent discussions about GitLab CI, do you think this repo might be a good (easy) starting point to explore GitLab CI? Since this repo only contains examples, all of which I'm guessing can run in unix Docker containers, it should be fairly easy to write a CI/CD pipeline specifically for this repo.

You need to check, but why not. FTR, CI for examples is NOT here, as documented, it is on the DeepSpeech repo side, because we want to run examples on the repo PRs, either on stable branches (`r0.8` at the moment) or on currently developped codebase (`master`) so we can know, when we do pushes to the main repo, whether we are breaking examples.

It should already be more or less explicit in the CI issue where we discussed about for GitLab, but don't hesitate if you need help.

Reply:
Alright, I'll take a look at the main repo's CI a bit later, thanks!